nucypher/blockchain/eth/domains.py

```diff
@@ -0,0 +1,70 @@
+from enum import Enum
+from typing import NamedTuple
+
+
+class ChainInfo(NamedTuple):
+    id: int
+    name: str
+
+
+class EthChain(ChainInfo, Enum):
+    MAINNET = (1, "mainnet")
+    GOERLI = (5, "goerli")
+    SEPOLIA = (11155111, "sepolia")
+    TESTERCHAIN = (131277322940537, "eth-tester")
+
+
+class PolygonChain(ChainInfo, Enum):
+    POLYGON = (137, "polygon")
+    MUMBAI = (80001, "mumbai")
+    TESTERCHAIN = (131277322940537, "eth-tester")
+
+
+class DomainInfo(NamedTuple):
```

Comment:
This namespace seems superfluous to me and reintroduces the same kind of repetition and indirection in previous iterations.  I do think a dictionary will reduce the line count and better-suited for the way you are accessing these values.  That being said It's not a blocker since this does work... just a bit bulky IMO.

Reply:
👍🏻 Ultimately you've got it down to 55 lines with much better semantics then before.  I can also appreciate *explicit is better than implicit*!  LGTM

Reply:
✔️